app/src/main/java/openfoodfacts/github/scrachx/openfood/features/allergensalert/AllergensAlertFragment.kt

```diff
@@ -219,7 +220,7 @@ class AllergensAlertFragment : NavigationBaseFragment() {
     }
 
     companion object {
-        private val LOG_TAG = AllergensAlertFragment::class.simpleName
+        private const val LOG_TAG = "AllergensAlertFragment"
```

Comment:
Why this?

Reply:
small optimization to not calculate the value in runtime and do it during compile time

Reply:
All other classes do this. Also, I don't think it gives us any benefit since it's a one-time operation and in case of a refactor it should be updated. If you aren't strongly suggesting this I would revert.

Reply:
reverted, I removed LOG_TAG at all because it was unused